image/tarexport/tarexport.go

```diff
@@ -30,18 +32,36 @@ type tarexporter struct {
 	loggerImgEvent LogImageEvent
 }
 
+type ocitarexporter struct {
+	te   *tarexporter
+	name string
+	refs map[string]string
+}
+
 // LogImageEvent defines interface for event generation related to image tar(load and save) operations
 type LogImageEvent interface {
 	//LogImageEvent generates an event related to an image operation
 	LogImageEvent(imageID, refName, action string)
 }
 
 // NewTarExporter returns new ImageExporter for tar packages
-func NewTarExporter(is image.Store, ls layer.Store, rs reference.Store, loggerImgEvent LogImageEvent) image.Exporter {
-	return &tarexporter{
+func NewTarExporter(is image.Store, ls layer.Store, rs reference.Store, loggerImgEvent LogImageEvent, format, name string, refs map[string]string) (image.Exporter, error) {
```

Comment:
I see no issues in having an options struct or moving this change to Save/Load API in tarexport ;) I think this can be worked out..